backend/cmd/headlamp.go

```diff
@@ -355,6 +449,133 @@ func createHeadlampHandler(config *HeadlampConfig) http.Handler {
 		http.Redirect(w, r, oauthConfig.AuthCodeURL(state), http.StatusFound)
 	}).Queries("cluster", "{cluster}")
 
+	r.HandleFunc("/startportforward", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Instead of using separate end points for `starting` and `deleting` portforward use a single endpoint like `/portforward` and use `r.Method()` and if the method is `POST` start a portforward and `GET` to list portforward and `DELETE` to delete the portforward.

Reply:
In `POST` method use the request body to get the request details instead of query params.

Reply:
In this implementation port forward is only supported for `pods` we will have to support `services`.

Reply:
changed this to /portforward post get and for list /portforward/list and for delete /portforward/delete